types/index.d.ts

```diff
@@ -289,10 +289,61 @@ declare namespace PhonegapPluginPush {
 		from?: string
 		notId?: string
 	}
-
+  /**
+   * Set Channel Variable,once created cannot be changed programmtically for importance and vibration
+   */
+  interface channel{
+    /**
+     * channel ID
+     */
+    id: string
+    /**
+     * camillebeaumont name and description fix
```

Comment:
Are these notes to fix later? We don't have the `name` attribute yet, so we don't have an official doc to copy from, so it would make sense.

Reply:
Yup based on a previous pull request